tests/ui/traits/next-solver/normalize/normalize-region-obligations.rs

```diff
@@ -15,7 +15,7 @@ trait Mirror { type Assoc: ?Sized; }
 impl<T: ?Sized> Mirror for T { type Assoc = T; }
 
 trait MirrorRegion<'a> { type Assoc: ?Sized; }
-impl<'a, T> MirrorRegion<'a> for T { type Assoc = T; }
+impl<'a, T: ?Sized> MirrorRegion<'a> for T { type Assoc = T; }
```

Comment:
The hrtb alias in the trait above is not wf b/c `Self` doesn't impl `MirrorRegion`. I think this was a typo when constructing this test.